components/Search/Hit.tsx

```diff
@@ -77,22 +93,80 @@ If neither the excerpt or the description is a match (like the match can be for
   a resonsible person) we will fallback to either the excerpt or description, like
   in the static text without any query)
 */
+
 const getSnippetAttribute = (hit: HitProps["hit"]) => {
   // eslint-disable-next-line no-underscore-dangle
   const excerptSnippet = hit._snippetResult?.excerpt as HitAttributeSnippetResult
   // eslint-disable-next-line no-underscore-dangle
   const descriptionSnippet = hit._snippetResult?.description as HitAttributeSnippetResult
+
   if (hit.excerpt && excerptSnippet && excerptSnippet.matchLevel === ("full" || "partial")) {
     return "excerpt"
     /* eslint no-else-return: "error"  */
   } else if (hit.description && descriptionSnippet && descriptionSnippet.matchLevel === ("full" || "partial")) {
     return "description"
   }
+
   return hit.excerpt ? "excerpt" : "description"
 }
 
+const isNewDataProduct = (hit: HitProps["hit"]) => {
+  const currentDate = new Date()
+  const compareDate = new Date(currentDate.setMonth(currentDate.getMonth() - 1))
+  const createdAtDate = new Date(hit.createdAt)
+  return createdAtDate.getTime() >= compareDate.getTime()
+}
```

Comment:
i don't think this should be calculated in the client. at least in its current state. this implementation will use the client's time zone, while the `createdAt` date is likely UTC, so people in different time zones will see different results. which for this use case isn't the biggest deal, but we should strive for every user to get presented the same results regardless of time zone imho

Reply:
additionally, if we implemented it in the indexer service, we would be able to let the users filter for it :+1:

Reply:
Do we have another date then createdTime? Since I don't think this is as accurate as we need? 🤔  Like country data, it has a create data several months ago even if it went live the other day